### DISCUSSION TITLE
Is there a way to ignore or reset previous version updates - needed for test purposes

### DISCUSSION THREAD

**Discussion by ernievd** [renovatebot/renovate#14898] [Request Help] [answered]
I am testing my new Renovate setup and have to delete the project I am using to test the updates on and create a new one every time I want to run the tests. This is because once Renovate updates a package to its latest version it will not create another MR it again even if I set the file back to the old version.

For example - 
In my Dockerfile I originally have :
`FROM openjdk:17-slim@sha256:a507bb03601e726027ad01ae3f540c8b936ec1ab75512767a4e2e732da4fe7e3`

I run Renovate and it creates an MR to update it to -
`FROM openjdk:17-slim@sha256:acec77a00db352e6552fb5929eb05c11ae0d4aca3441e8dd48dc42820c5ab108`

I approve the MR and it merges fine.

I want to test again so in the same project I update the Dockerfile to once again have -
`FROM openjdk:17-slim@sha256:a507bb03601e726027ad01ae3f540c8b936ec1ab75512767a4e2e732da4fe7e3`

I rerun Renovate and it does not create an MR to update the Dockerfile to the acec77 hash like it did earlier. It ignores it now.

I assume Renovate keeps some sort of log or history that it checks against that causes this behavior?

Is there anyway to prevent this from happening so I do not have to create a new project every time?

I renamed the Dockerfile but that did not change anything.

I also created a new file(named Dockerfile.2) in the same project with the old hash and that also did not work.

 Is there was a way to reset this as to ignore what happened in the past.

**Comment by rarkins** [comment, marked as answer]
Set recreateClosed=true in config

**Reply by ernievd** [reply]
Thanks so much!
